### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by ghost**
Tests for products_aggregator.

### CODE REVIEW

File: apps/main/admin.py
Comment (sibiryoff) [inline apps/main/admin.py]: ??? Why it is changed within this task?

File: apps/main/tests/test_models.py
Comment (sibiryoff) [inline apps/main/tests/test_models.py]: Do not place test file in MEDIA_ROOT folder.

File: apps/main/tests/test_views.py
Comment (sibiryoff) [inline apps/main/tests/test_views.py]: Same as in previous comment.

File: apps/main/tests/test_views.py
Comment (sibiryoff) [inline apps/main/tests/test_views.py:117]: What about access for non-Superviser users?

**Comment by sibiryoff** [comment]
Great work. Let's focus on comments before final approval.

**Comment by ghost** [comment]
On this moment tests failed, because pull request bugfix/08 don't merge into master.

**Comment by sibiryoff** [comment]
@leocoan please make sure that all tests can be run by `python manage.py test` command.

**Comment by ghost** [comment]
@sibiryoff now command `python manage.py test` run all test in the all application. Now we have one application - apps.main.

**Review by sibiryoff — APPROVED** [review]

**Pull request merged**
